src/components/header.tsx

```diff
@@ -105,8 +107,14 @@ function Header(props: HeaderProps) {
           >
             <Logo width="54" height="54" />
           </Link>
+          <div className="relative">
+            <Search className="absolute left-3 top-1/2 transform -translate-y-1/2 text-[#737373] w-4 h-4" />
+            <Input
+              placeholder="Search"
+              className="pl-10 bg-[#262626] border-[#404040] text-white placeholder-[#737373] w-80"
```

Comment:
use themed color name instead of hex value. you can check it in global.css. 
If absence, you can define color name based on figma.